README.md

```diff
@@ -103,162 +103,4 @@ The latest versions of all BOM lines are available from the [Jenkins artifact re
-
-To further minimize build time, tests are run only on Linux, against Java 11, and without Docker support.
-It is unusual but possible for cross-component incompatibilities to only be visible in more specialized environments (such as Windows).
-
-## LTS lines
-
-A separate BOM artifact is available for the latest weekly, current LTS line and a few historical lines.
-BOMs should only specify plugin version overrides compared to the next-newer BOM.
-`sample-plugin` will use the weekly line by default,
-and get a new POM profile for the others.
-To get ahead of problems, prepare the draft PR for a line as soon as its baseline is announced.
-
-The CI build (or just `mvn test -P2.nnn.x`) will fail if some managed plugins are too new for the LTS line.
-[This script](https://gist.github.com/jglick/0a85759ea65f60e107ac5a85a5032cae)
-is a handy way to find the most recently released plugin version compatible with a given line,
-according to the `jenkins-infra/update-center2`.
-
-The [developer documentation](https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/) recommends the last releases of each of the previous two LTS baselines.
-BOMs for the current LTS release and two prior LTS releases are typically retained.
-BOMs older than the two prior LTS releases will generally be retired in order to better manage evaluation costs and maintenance efforts.
-
-## Releasing
-
-You can cut a release using [JEP-229](https://jenkins.io/jep/229).
-To save resources, `master` is built only on demand, so use **Re-run checks**  in https://github.com/jenkinsci/bom/commits/master if you wish to start.
-If that build passes, a release should be published automatically when PRs matching certain label patterns are merged.
-For the common case that only lots of `dependencies` PRs have been merged,
-the release can be triggered manually from the **Actions** tab after a `master` build has succeeded.
-
-## Incrementals
-
-This repository is integrated with “Incrementals” [JEP-305](https://jenkins.io/jep/305):
-
-* Individual BOM builds, including from pull requests, are deployed and may be imported on an experimental basis by plugins.
-  (The plugin’s POM must use the `gitHubRepo` property as shown in [workflow-step-api-plugin #58](https://github.com/jenkinsci/workflow-step-api-plugin/pull/58/files).)
-* Pull requests to the BOM may specify incremental versions of plugins, including unmerged PRs.
-  (These should be resolved to formal release versions before the PR is merged.)
-
-Together these behaviors should make it easier to verify compatibility of code changes still under review.
-
-## GitHub tooling
-
-This repository uses Dependabot to be notified automatically of available updates, mainly to plugins.
-(It is not currently possible for Jenkins core updates to be tracked this way.)
-
-Release Drafter is also used to prepare changelogs for the releases page.
+[Moved to CONTRIBUTING](CONTRIBUTING.md)
```

Comment:
Was there any textual change besides this move?

Reply:
No. There will be another PR once merged where all the "work" will be done. I wanted to keep the break clean.